lib/ipfs-pinner.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+const path = require('path')
+const { client, PinnerError } = require('@aragonone/ipfs-pinner-client')
+
+const ALREADY_UPLOADED_ERROR = 'File is already uploaded with cid '
+const IPFS_PINNER_ENDPOINT = 'https://ipfs-pinner.backend.aragon.org'
+
+function requiresIpfsUpload(justification) {
+  return !justification.startsWith('ipfs:') && justification.endsWith('.md')
+}
+
+async function ipfsUpload(disputePath, justification, sender) {
+  let content
+
+  if (justification.endsWith('.md')) {
+    const justificationPath = path.join(process.cwd(), disputePath, justification)
+    if (!fs.existsSync(justificationPath)) throw Error(`Justification path "${justificationPath}" does not exit`)
+    console.log(`Uploading justification "${justification}" to IPFS...`)
+    content = justificationPath
+  } else {
+    console.log(`Uploading justification "${justification}" as plain text (only markdown is supported for IPFS evidence)...`)
```

Comment:
Not sure I get this; don't we try to upload `content` to IPFS regardless after this `else`?

Reply:
we are, but in this case we are uploading it as plain text